src/runtime/wasm.ts

```diff
@@ -31,6 +31,9 @@ import {Dictionary} from './hot.js';
 // Examples:
 //   Singleton:   4:id05|txt:T3:abc|lnk:U10:http://def|num:N37:|flg:B1|
 //   Collection:  3:29:4:id12|txt:T4:qwer|num:N9.2:|18:6:id2670|num:N-7:|15:5:id501|flg:B0|
+//
+// The encoder classes also support a "Dictionary" format of key:value string pairs:
+//   <size>:<key-len>:<key><value-len>:<value><key-len>:<key><value-len>:<value>...
```

Comment:
Can we format this string as a jsdoc? i.e. `/** ... */`?

Reply:
It will help the generation of https://live.arcs.dev/dist/apidocs/

Reply:
I'll do that in a separate CL, to avoid muddying the logical changes here.

Reply:
Actually, not really sure of the utility for that. Having generated docs for internal classes doesn't seem all that useful - I certainly just look at the actual code for details on APIs.